charts/ocis/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
   repository: owncloud/ocis
-  tag: "1.17.0"
-  sha: ""
+  tag: "latest"
+  sha: "a49a106802ebe44e1cfdc04ca71661be104d87224f0e6fd8e2477bc3bb078b92"
```

Comment:
Is it necessary to pin the SHA here? Shouldn't we rather wait for an explicit version with the needed fixes?